### ISSUE TITLE
Merge commits

### ISSUE THREAD

**Issue by Ironbatpug** [uwolfer/gerrit-intellij-plugin#402]
I do not know if it is a bug or a question. Maybe I have mis configured something, but this is my problem.
I installed this plugin to idea, and set up everything. I am seeing the changes and a differences, but at some changes I only see the commit message. I have opened the change in browser and I have seen many file change. The change was a merge commit. Does this plugin show merge commits as the browser does?

BTW the plugin is fantastic.
Thank you

**Comment by uwolfer** [comment]
You are right, merge commits do not show up in the diff browser the same way as they do in the web UI.

Reason: for the diff browser in IntelliJ, we use the IntelliJ diff implementation, which does not display all changes included in this merge commit. You can see the same behavior in IntelliJ Git tooling log view for merge commits.

Gerrit does some magic for merge commits.

Changing this behavior is probably not that easy - at least I do not know a way to do so. We could probably using the Gerrit REST API for fetching the diff of a merge, but that would involve heavy changes in the plugin implementation.

But I agree, the merge diffing in Gerrit is very nice and also useful.
